docs/examples/iq_plot_listen.py

```diff
@@ -0,0 +1,259 @@
+        for sequence in range(sequences_to_play):
+            iq_real = np.array([x[0] for x in data[sequence][0]])
+            iq_imag = np.array([x[1] for x in data[sequence][0]])
+            # TODO: Stereo sound with real and imag in the two channels
+            signals.extend(iq_real)
+
+    # TODO: Don't hardcode the sampling rate at 3333
+    soundfile.write(wav_filename, signals, 3333, 'PCM_16')
+
+
+def plot_iq(records, save_directory, sequences_to_plot=None):
+    """
+    Plot the records in a nice way.
+
+    That's nice. Optionally specify up to how many sequences
+    to plot for each record. It will plot them in order up to the number specified or all
+    of the sequences, whichever is less.
+    :param records: pydarn iqdat records, as returned from read_iqdat()
+    :param save_directory: Filepath to save the plots to
+    :param sequences_to_plot: up to how many sequences should be plotted for each record?
+    :return:
+    """
+    # TODO: Print out progress bar, because this is slow.
+    for record in records:
+        stid = record['stid']
+        radar_abbrev = pydarn.SuperDARNRadars.radars[stid].hardware_info.abbrev
+        record_datetime = dt.datetime.strptime(record['origin.time'], "%a %b %d %H:%M:%S %Y")
+        record_datetime = record_datetime.strftime("%Y%m%d.%H%M.%S")
+        plot_filename = save_directory + '/' + record_datetime + '.' + radar_abbrev + \
+                        '.bm' + str(record['bmnum']) + '.png'
+
+        if sequences_to_plot is None:
+            # Plot all of them!
+            sequences_to_plot = int(record['seqnum'])
+        if not isinstance(sequences_to_plot, int):
+            print("Need to specify an integer for sequences_to_plot, plotting all.")
+            sequences_to_plot = record['seqnum']
+        sequences_to_plot = max(sequences_to_plot, record['seqnum'])
+
+        # Need to know how many samples per tau (what samples could contain a tx pulse?)
+        samps_per_tau = record['mpinc'] / record['smsep']
+        # Determine which samples overlap with tx pulses
+        blanked_samples = [record['skpnum'] - 1 + pulse * samps_per_tau for pulse in record['ptab']]
+
+        fig = plt.figure(figsize=(11, 8.5))  # Regular 8.5" x 11" sheet of paper size
+        figtop = 0.88
+        figheight = 0.82 / sequences_to_plot
+        data = np.copy(record['data'])
+
+        data = data.reshape((int(record['seqnum']), int(record['chnnum']), int(record['smpnum']), 2))
+
+        for sequence in range(sequences_to_plot):
+            # Calculate positions of the data axis and colorbar axis for the current parameter
+            # and then add them to the figure. position is [left, bottom, width, height] in fractions
+            # of figure width and height
+            position = [0.1, figtop - figheight * (sequence + 1) + 0.05, 0.8, figheight]
+            ax = fig.add_axes(position)
+
+            if sequence is 0:
```

Comment:
```suggestion
            if sequence == 0:
```
Is sequence an integer?
`is` is only used on objects and str types.

Reply:
cool I had no idea, thanks! I'm updating that along with other things like stereo and docs.